experimental/dds/tree2/src/feature-libraries/editable-tree/editableField.ts

```diff
@@ -136,6 +158,37 @@ export class FieldProxyTarget extends ProxyTarget<FieldAnchor> implements Editab
 		return this.cursor.getFieldLength();
 	}
 
+	/* eslint-disable no-param-reassign */
+
+	public slice(start?: number, end?: number) {
```

Comment:
Adding a method here, but not in the interface this type implements does not result in this method being added to the API. You should make your API change to src/feature-libraries/editable-tree/editableTreeTypes.ts,  includ documentation for what it does (and what it returns) there.

Also we generally avoid implicit return types: it makes code (especially undocumented functions) hard to reason about. I'd like to have a documented function on the interface with a return type before I evaluate if this is a good API. its hard to tell what it does right now.

Reply:
Do you want array-like APIs exposed via the EditableField API, or only accessible via the proxy?